src/main/java/com/google/devtools/build/lib/syntax/MethodLibrary.java

```diff
@@ -221,41 +221,38 @@ private static boolean hasElementWithBooleanValue(
     } else {
       checkValidKeyFunc(key, loc);
 
-      final Object KEY = key;
-      final Location LOC = loc;
-      final Environment ENV = env;
-      final FuncallExpression AST = new FuncallExpression(Identifier.of(""), ImmutableList.of());
+      final FuncallExpression ast = new FuncallExpression(Identifier.of(""), ImmutableList.of());
 
       class KeyComparator implements Comparator<Object> {
-        private Exception e;
-
-        public Exception getException() {
-          return this.e;
-        }
+        public Exception e;
 
         @Override
         public int compare(Object o1, Object o2) {
           try {
             return EvalUtils.SKYLARK_COMPARATOR.compare(
-                    evalKeyFunc(o1, KEY, LOC, ENV, AST), evalKeyFunc(o2, KEY, LOC, ENV, AST));
-          } catch (Exception e) {
-            this.e = this.e == null ? e : this.e;
+                    evalKeyFunc(o1, key, loc, env, ast),
+                    evalKeyFunc(o2, key, loc, env, ast));
+          } catch (InterruptedException | EvalException e) {
+            if (this.e == null) {
+              this.e = (Exception) e;
+            }
+            return 0;
           }
-          return 0;
         }
       }
 
-      KeyComparator comparator = new KeyComparator();
-      Collections.sort(list, comparator);
+      KeyComparator comp= new KeyComparator();
```

Comment:
space before '='

Reply:
fyi, spaces and indentation will be fixed automatically when we import the change (we run a formatter)